app/lib/three_scale/redis_config.rb

```diff
@@ -56,5 +58,16 @@ def parse_sentinels(sentinels)
         parsed_sentinel
       end
     end
+
+    def load_default_ca_cert(conf)
+      return if conf[:ssl_params]&.key?(:ca_file) || conf[:ssl_params]&.key?(:ca_path)
```

Comment:
Presently we have in config
```
ca_file: <%= ENV['BACKEND_REDIS_CA_FILE'] %>
```

This (I believe) means that the keys will always be set and the default will never be used. At least it requires testing. I expect that we would need something like
```suggestion
      return if conf.dig(:ssl_params, :ca_file).presence? || conf.dig(:ssl_params, :ca_path).presence?
```
It will still require some manual testing though to make sure behavior is as we expect.

Reply:
Good catch, it was actually failing in that situation, I fixed it and added a new test for this.